setup.cfg

```diff
@@ -81,6 +81,7 @@ metrics =
     sacrebleu~=2.2.1  # For disinformation_metrics, machine_translation_metrics
     langdetect~=1.0.9  # For ifeval_metrics
     immutabledict~=4.2.0  # For ifeval_metrics
+    gradio_client==1.4.3  # For bigcodebench_metrics
```

Comment:
Change to `gradio_client~=1.3`

`gradio_client~=1.4.3` isn't supported by Python 3.9.